slo_generator/backends/prometheus.py

```diff
@@ -97,7 +98,12 @@ def good_bad_ratio(self, timestamp, window, slo_config):
         return (good_count, bad_count)
 
     # pylint: disable=unused-argument
-    def distribution_cut(self, timestamp, window, slo_config):
+    def distribution_cut(
+        self,
+        timestamp: int,
+        window: int,
+        slo_config: dict
+    ) -> tuple[float, float]:
```

Comment:
Are you sure about the `tuple[float, float]` return type?

Reply:
Yes sure.

The `good_count` is coming from `count()` which returns float [line 177]
And the `bad_count` is coming from a sub of two floats coming of from `count()` which returns float [line 177]